src/copyright/ui/ajax-copyright-hist.php

```diff
@@ -367,6 +367,17 @@ private function getOrderString()
     return $orderString;
   }
 
+  /**
+   * @brief get Agent ID for an upload
+   * @param Integer $upload_pk Upload ID
+   * @param string $copyrightType Copyright Type
+   * @return Integer Agent ID
+   */
+  public function getAgentId($upload_pk,$copyrightType)
+  {
+    return(LatestAgentpk($upload_pk, $copyrightType));
+  }
```

Comment:
Please follow good coding style. Check guideline at: https://github.com/fossology/fossology/wiki/Coding-Style#coding-guidelines-for-php
```suggestion
   * @brief Get Agent ID for an upload
   * @param integer $upload_pk Upload ID
   * @param string $copyrightType Copyright Type
   * @return integer Agent ID
   */
  public function getAgentId($upload_pk,$copyrightType)
  {
    return LatestAgentpk($upload_pk, $copyrightType);
  }
```